### ISSUE TITLE
Doubts about Aux model path

### ISSUE THREAD

**Issue by rcdlei7** [TJU-sjyj/AMU-Tuning#3]
Hi, first of all thank you very much for your excellent work, I would like to ask if the auxiliary model pre-training model file is the moco-v3 version? Also about the training files for the 11 downstream tasks can you post them? I'm using the tip-adapter dataset setup as described and it works but the results are not quite as expected, if so please contact me. Email: 2661067683@qq.com

**Comment by fly361** [comment]
I also meet this problem, the results are not quite as expected, Did you solve this problem?

**Comment by Leg-end** [comment]
I also meet same problem, I used moco-v3/r-50-1000ep.pth.tar as auxiliary model. I conducted many versions of experiments. The following is my results on ImageNet with RN50 backbone (learning rate=1e-3, train_epoch=50, bactch_size=8)

|Arguments                                                                                                                                             |1-shot|2-shot|4-shot|8-shot|16-shot|
|------------------------------------------------------------------------------------------------------------------------------------------------------|------|------|------|------|-------|
|augment_epoch=1 <br> alpha=0.5 <br> lambda_merge=0.35 <br> uncent_type=none <br> seed=2                                                               |59.81 |63.21 |65.60 |68.12 |69.79  |
|augment_epoch=1 <br> alpha=0.5 <br> lambda_merge=0.4 <br> uncent_type=moment <br> uncent_power=0.4 <br> seed=1                                        |61.22 |63.65 |65.65 |67.65 |69.34  |
|augment_epoch=1 <br> alpha=0.5 <br> lambda_merge=0.35 <br> uncent_type=none <br> seed=2                                                               |-     |-     |-     |-     |69.27  |
|augment_epoch=10 <br> alpha=0.5 <br> lambda_merge=0.35 <br> uncent_type=none <br> seed=2                                                              |-     |-     |-     |-     |69.79  |
|augment_epoch=1 <br> alpha=0.5 <br> lambda_merge=0.35 <br> uncent_type=none <br> seed=3                                                               |-     |-     |-     |-     |69.88  |
|augment_epoch=1 <br> alpha=0.5 <br> lambda_merge=0.35 <br> uncent_type=none <br> seed=4                                                               |-     |-     |-     |-     |69.18  |
|Evaluate using pretrained adapter <br> augment_epoch=1 <br> alpha=0.5 <br> lambda_merge=0.35 <br> uncent_type=none <br> seed=2                        |-     |-     |-     |-     |69.66  |
|Evaluate using pretrained adapter <br> augment_epoch=1 <br> alpha=0.5 <br> lambda_merge=0.35 <br> uncent_type=moment <br> uncent_power=0.4 <br> seed=2|-     |-     |-     |-     |68.91  |

Here's what now I conclude
1. best test accuray of 1-shot reach at 1-epoch, looks like augment plays no effec
2. uncertainty logit fusion seems work at 1, 2, 4 shot, but harms accuray at 8, 16 shot
3. Also, from supplement materail of paper, best lambda merge and uncent power should both be 0.4
4. Anyway, I can not reproduce the results in paper, and my issue in this repository never responsed
5. I also tried to evaluate using pretrained adapter weight, but the result still not match the one in paper

Have you meet similar issue in the process of reproduction ? if so please contact me in the repo or Email: 231027163@fzu.edu.cn
